pennylane/_qubit_device.py

```diff
@@ -312,19 +312,23 @@ def sample_basis_states(self, number_of_states, state_probability):
         return np.random.choice(basis_states, self.shots, p=state_probability)
 
     @staticmethod
-    def states_to_binary(samples, num_wires):
+    def states_to_binary(samples, num_wires, dtype=None):
```

Comment:
It perhaps might make sense to place the default dtype here?
```suggestion
    def states_to_binary(samples, num_wires, dtype=np.int64):
```